pages/ekman.js

```diff
@@ -0,0 +1,282 @@
+            <option value="en">english</option>
+            <option value="de">german</option>
+            <option value="es">espanol</option>
+          </select>{" "}
+        </label>
+        <p>{question[page] && question[page]}</p>
+        {page === 10 && (
+          <p>
+            <TbList onClick={() => setShowList(!showList)} />
+            <span onClick={() => setShowList(!showList)}>
+              {showList ? "List-View" : "Circles-View"}
+              {"   "}
+            </span>
+          </p>
+        )}
+        {page === 0 && (
+          <StyledTextarea
+            type="text"
+            name="0"
+            value={formData["0"]}
+            onChange={handleChange}
+            placeholder="write here..."
+            required
+          />
+        )}
+        {page === 1 && (
+          <StyledTextarea
+            type="text"
+            name="1"
+            value={formData["1"]}
+            onChange={handleChange}
+            placeholder="write here..."
+          />
+        )}
+        {page === 2 && (
+          <StyledTextarea
+            type="text"
+            name="2"
+            value={formData["2"]}
+            onChange={handleChange}
+            placeholder="write here..."
+          />
+        )}
+        {page === 3 && (
+          <StyledTextarea
+            type="text"
+            name="3"
+            value={formData["3"]}
+            onChange={handleChange}
+            placeholder="write here..."
+          />
+        )}
+        {page === 4 && (
+          <StyledTextarea
+            type="text"
+            name="4"
+            value={formData["4"]}
+            onChange={handleChange}
+            placeholder="write here..."
+          />
+        )}
+        {page === 5 && (
+          <StyledTextarea
+            type="text"
+            name="5"
+            value={formData["5"]}
+            onChange={handleChange}
+            placeholder="write here..."
+          />
+        )}
+        {page === 6 && (
+          <StyledTextarea
+            type="text"
+            name="6"
+            value={formData["6"]}
+            onChange={handleChange}
+            placeholder="write here..."
+          />
+        )}
+        {page === 7 && (
+          <StyledTextarea
+            type="text"
+            name="7"
+            value={formData["7"]}
+            onChange={handleChange}
+            placeholder="write here..."
+          />
+        )}
+        {page === 8 && (
+          <StyledTextarea
+            type="text"
+            name="8"
+            value={formData["8"]}
+            onChange={handleChange}
+            placeholder="write here..."
+          />
+        )}
+        {page === 9 && (
+          <>
+            <StyledTextarea
+              type="text"
+              name="9"
+              value={formData["9"]}
+              onChange={handleChange}
+              placeholder="write here..."
+            />
+          </>
+        )}
```

Comment:
This feels very repetitive. Maybe we could clean it up a little. Here is one possible suggestion. 

```suggestion
     {page >= 0 && page <= 9 && (
        <StyledTextarea
          type="text"
          name={page.toString()}
          value={formData[page.toString()]}
          onChange={handleChange}
          placeholder="write here..."
          required={page === 0}
        />
      )}
```